ietf-document/draft-0rtt-bdp/draft-kuhn-quic-0rtt-bdp-07.xml

```diff
@@ -72,56 +72,214 @@
+			the server refuses the optimization.</t>
 		</section>
 
 		<section anchor="sec:other_parameter" title="Discussion">
-			<t>The recon_bytes_in_flight parameter is higher than the actual BDP since it may include bytes in buffers along the path. The recon_bytes_in_flight parameter is still a indication of the end-to-end BDP that is experienced by the congestion control. If the recon_bytes_in_flight is high, the server may decide to increase the maximum amount of packets it will send when reconnecting using the recon_max_pkt_number parameter. The maximum number of initial data packets that can be sent without acknowledgement needs to be chosen to avoid congestion collapse. An example of a server solution is proposed in <xref target="appendix-server"></xref>. In short:<list style="symbols">
-				<t>recon_bytes_in_flight indicates the characteristics of the network underneath to both peers that can adapt their buffer sizes or parameter tuning.</t>
-				<t>recon_min_rtt lets both client and server know the minimum RTT of the previous connection. Parameters can then be adapted (e.g. adapt how kInitialRtt is used in the "Setting the Loss Detection Timer" section of <xref target="I-D.ietf-quic-recovery"></xref>).</t>
-				<t>recon_max_pkt_number lets the server warn the client that it may increase the amount of packets that it expects to send when reconnecting. This value is negotiated with the client and result in better buffer management and reduced flow start up.</t></list></t>
+			<t>The recon_bytes_in_flight parameter is higher than
+			the number of bytes in the actual BDP since it may
+			include bytes in buffers along the path. </t>
+
+			<t>The recon_bytes_in_flight parameter is
+			an indication of the end-to-end BDP that is
+			experienced by the congestion control. If the
+			recon_bytes_in_flight is high, the server may decide to
+			increase the maximum amount of packets it will send
+			when reconnecting using the recon_max_pkt_number
+			parameter. </t>
+
+			<t>The maximum number of initial data packets
+			that can be sent without acknowledgement needs to be
+			chosen to avoid congestion collapse. An example of a
+			server solution is proposed in <xref
+			target="appendix-server"></xref>. In short:
+			<list style="symbols">
+				<t>recon_bytes_in_flight indicates the
+				characteristics of the network underneath to
+				both peers that can adapt their buffer sizes or
+				parameter tuning.</t>
+
+				<t>recon_min_rtt lets both client and server
+				know the minimum RTT of the previous
+				connection. Parameters can then be adapted
+				(e.g. adapt how kInitialRtt is used in the
+				"Setting the Loss Detection Timer" section of
+				<xref
+				target="I-D.ietf-quic-recovery"></xref>).</t>
+
+				<t>recon_max_pkt_number lets the server warn
+				the client that it may increase the amount of
+				packets that it expects to send when
+				reconnecting. This value is negotiated with the
```

Comment:
is this speaking about flow control or CC or both? ... I think we should be clearer.